docs/conf/helpop-full.conf.example

```diff
@@ -434,11 +434,11 @@ reason parameter is optional, and if not provided defaults to
 'Please use this server/port instead' (the default given in various
 numeric lists)">
 
-<helpop key="filter" value="/FILTER <filter-definition> [<action> <flags> [<gline-duration>] :<reason>]
+<helpop key="filter" value="/FILTER <filter-definition> [<action> <flags> [<duration>] :<reason>]
 
 This command will add a filter when more than one parameter is given,
 for messages of the types specified by the flags, with the given
-filter definition, action, gline duration (when the action is 'gline')
+filter definition, action, duration (only for 'gline' and 'shun'),
```

Comment:
>filter definition, action, and duration (when the action is 'gline' or 'shun'),

Reply:
The list continues onto the next line ("and reason"), which is why I won't put an "and" before duration. I'll change the text in parentheses though.

Reply:
Thanks for clarifying, it would be nice if GitHub diffs showed that. 😐